tests/common/util.rs

```diff
@@ -2548,9 +2548,12 @@ mod tests {
     fn test_uchild_when_run_no_wait_with_a_non_blocking_util() {
         let ts = TestScenario::new("echo");
         let mut child = ts.ucmd().arg("hello world").run_no_wait();
-        child.delay(500);
 
-        // check `child.is_alive()` is working
+        // check `child.is_alive()` and `child.delay()` is working
+        while child.is_alive() {
```

Comment:
Do we maybe want to limit this if the process is unresponsive? E.g. try 10 times and then panic?

Reply:
although I would have preferred to set a timeout ;)